src/collections/_documentation/guides/discover/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Uncover Trends with Discover
+sidebar_order: 5
+---
+
+[Discover]({%- link _documentation/performance/discover/index.md -%}) is a powerful query engine that allows you to query through all your error metadata across projects and applications. As you've probably noticed the Sentry SDKs installed in your applications capture huge amounts of events as reflected in the event stream in your Sentry account. Each event is and enriched with contextual data about the underlying error, platform, device, user, and more. Also, we highly encourage enriching your error data through the SDK by adding custom tags and configuring the release and environment. For more information see [Put your Data to Work]({%- link _documentation/guides/enrich-data/index.md -%}).
```

Comment:
Do you want to use the updated linking structure (note for throughout the file)?

"query through" reads awkwardly. I'd suggest "query all your error data across"

"Each event is enriched with" (remove the "and")